### PR TITLE
WIP: Rehearsals for NetObserv testing

### PR DESCRIPTION
**Pull request by Amoghrd**

### CODE REVIEW

**Comment by Amoghrd** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-automated-release-gcp-ipi-mini-perm-custom-type-f360

**Comment by memodi** [comment]
@Amoghrd you want have changes like this PR https://github.com/openshift/release/pull/56883/files . The current changes will run all other tests and will take very long.

**Comment by Amoghrd** [comment]
As of now its selecting all level0 tests but I made a mistake in the TEST_FILTERS line above. Should have removed everything and only kept Network_Observability

File: ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ chain:
   - ref: openshift-extended-test
   - ref: cucushift-e2e
   - ref: openshift-extended-test-supplementary
+  - ref: wait
+    env:
+    - name: SLEEP_DURATION
+      default: "5m"
```
Comment (memodi) [inline ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:24]: you want something longer wait for couple of hours to be able to debug

**Comment by Amoghrd** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-automated-release-gcp-ipi-mini-perm-custom-type-f360

**Pull request closed without merging**